conf/layer.conf

```diff
@@ -1,5 +1,6 @@
 BBPATH .= ":${LAYERDIR}"
 
+BBFILES += "${LAYERDIR}/recipes-smt/*/*.bb"
```

Comment:
It does not look like there is a very large number of SMT solvers that are supported by yosys-smtbmc. I am thinking it is probably worth creating/locating these in `recipes-support` instead, that way there is a good location in the layer for recipes that are supporting a tool/workflow, but that are themselves not a HDL specific tool/library.

Reply:
Good suggestion, will do.